lib/bummr/git.rb

```diff
@@ -0,0 +1,18 @@
+module Bummr
+  class Git
+    include Singleton
+    include Log
+
+    def initialize
+      @git_commit = ENV.fetch("BUMMR_GIT_COMMIT") { "git commit" }
+    end
+
+    def commit(message)
+      system("#{git_commit} -am '#{message}'")
```

Comment:
Can we make it just `git commit -m"#{message}"` and drop the `a` ?

Probably better to do the adding and the committing separately if we're going to use this kind of pattern.

Reply:
For example, you can see that in my latest PR, I only add `Gemfile` and `Gemfile.lock` (I noticed sometimes it was committing logs that were generated)

https://github.com/lpender/bummr/pull/51/files#diff-dd5832feaa364e94da7df94cd3c2f8f6R39